scripts/plotting/reprocessed_gatk4_samples_pca/README.md

```diff
@@ -0,0 +1,10 @@
+# Perform pca on the three 1kG reprocessed GATK4 samples
+
+This runs a Hail query script in Dataproc using Hail Batch in order to plot a PCA of the three 1kG samples reprocessed using the GATK4 pipeline. To run, use conda to install the analysis-runner, then execute the following command:
+
+```sh
+POP=nfe
+analysis-runner --dataset tob-wgs \
+--access-level standard --output-dir "gs://cpg-tob-wgs-analysis/1kg_hgdp_${POP}/v0" \
```

Comment:
Can we store this in the `web` bucket instead, for easier viewing?

Reply:
yeah, definitely! So that would be "gs://cpg-tweb-analysis/1kg_hgdp_${POP}/v0"?

Reply:
Instead of `analysis`, it would be `web`, i.e. `cpg-tob-wgs-web`.